README.md

```diff
@@ -106,19 +106,19 @@ source <name of virtual environment>/bin/activate
 
 ### Initializing Keep
 ```
-init_plenum_raet_keep --name Alpha --seeds 000000000000000000000000000Alpha Alpha000000000000000000000000000 --force
+init_plenum_keys --name Alpha --seeds 000000000000000000000000000Alpha Alpha000000000000000000000000000 --force
```

Comment:
Have we updated Getting Started and similar scripts (both written for humans, and bash scripts) that call init_plenum_raet_keep?